zerver/views/streams.py

```diff
@@ -113,6 +113,16 @@ def list_subscriptions_backend(request, user_profile):
     # type: (HttpRequest, UserProfile) -> HttpResponse
     return json_success({"subscriptions": gather_subscriptions(user_profile)[0]})
 
+@has_request_variables
+def update_subscriptions_property(request, user_profile, stream_name=REQ(), property=REQ(), value=REQ()):
+    # type: (HttpRequest, UserProfile, str, str, str) -> HttpResponse
+    (stream, recipient, sub) = access_stream_by_name(user_profile, stream_name)
+    if sub is None:
+        return json_error(_("Not subscribed to stream %s") % (stream_name,))
+    do_change_subscription_property(user_profile, sub, stream,
+                                    property, ujson.loads(value))
+    return json_success()
```

Comment:
I guess the existing endpoint is a bulk endpoint, and this just does one?  So maybe we move the existing bulk-update one to  `POST /subscriptions/properties`

And the new simpler endpoint for just one should be:
`PATCH /users/me/subscriptions/<stream_id>`